bazel/wasm/wasm.bzl

```diff
@@ -73,15 +73,10 @@ def wasm_cc_binary(name, tags = [], **kwargs):
     kwargs.setdefault("visibility", ["//visibility:public"])
     cc_binary(
         name = wasm_name,
-        # Adding manual tag it won't be built in non-Wasm (e.g. x86_64 config)
-        # when an wildcard is specified, but it will be built in Wasm configuration
-        # when the wasm_binary below is built.
-        tags = ["manual"],
```

Comment:
Don't remove this manual tag, the comment is valid, but in this repo `.bazelrc` have emscripten toolcahin and cpu=wasm32 so it wont fail in CI, it will fail when you import this in Envoy.

Reply:
Yeah I kept it at the beginning, but removed it becauseI think usage of this rule will mostly be in wasm only repo, so it won't be an issue there. How about we keep the envoy_wasm_cpp_binary rule where we pass in manual tag?

Reply:
If you have a non-manual target depends on a manual target, then the manual target will be build, that's why this macro is defined (i.e. having a manual tagged `cc_binary` and use transition to enforce it built in wasm toolchain). If you are in a wasm only repo then you don't even need this rule and you should just use a naked `cc_binary`.

Reply:
Ok thanks! I think most cpp extension repo actually would have mix usage of tool chains because unit test will not use wasm toolchain, and it would be easier to just enforce manual build here.